include/dmlc/parameter.h

```diff
@@ -310,7 +310,7 @@ namespace parameter {
 class FieldAccessEntry {
  public:
   FieldAccessEntry()
-      : has_default_(false) {}
+      : has_default_(false), index_(0) {}
```

Comment:
Why are we adding a new field `index_` here?

Reply:
We aren't, just initializing it.